src/biome/text/explore.py

```diff
@@ -424,11 +430,18 @@ def add_predictions(batch, columns):
         num_proc=options.num_proc,
     )
 
+    # Quick fix for in-memory data that are not backed up by a file
+    # TODO: Finde a better solution
```

Comment:
Typo from my side ...
```suggestion
    # TODO: Find a better solution
```